cypress/e2e/users/custom-fields/verify-final-custom-field-removed.cy.js

```diff
@@ -0,0 +1,65 @@
+import getRandomPostfix from '../../../support/utils/stringTools';
+import { DevTeams, TestTypes, Permissions } from '../../../support/dictionary';
+import CustomFields from '../../../support/fragments/settings/users/customFields';
+import TopMenu from '../../../support/fragments/topMenu';
+import Users from '../../../support/fragments/users/users';
+import SettingsPane from '../../../support/fragments/settings/settingsPane';
+
+describe('Custom Fields', () => {
+  let user;
+  const fieldData = {
+    fieldLabel: `autotestFieldLabel_${getRandomPostfix()}`,
+    helpText: `autotestHelpText_${getRandomPostfix()}`,
+  };
+
+  const singleSelectData = {
+    data: {
+      fieldLabel: `autotestSingleSelectLabel_${getRandomPostfix()}`,
+      helpText: `autotestHelpText_${getRandomPostfix()}`,
+      firstLabel: `autotestFirstLabel_${getRandomPostfix()}`,
+      secondLabel: `autotestSecondLabel_${getRandomPostfix()}`,
+    },
+  };
+  const checkboxData = {
+    fieldLabel: `autotestFieldLabel_${getRandomPostfix()}`,
+    helpText: `autotestHelpText_${getRandomPostfix()}`,
+  };
+  before('Create test data', () => {
+    cy.getAdminToken();
+    cy.createTempUser([Permissions.uiUsersCustomField.gui, Permissions.uiUserEdit.gui]).then(
+      (userProperties) => {
+        user = userProperties;
+        cy.login(user.username, user.password, {
+          path: TopMenu.settingsPath,
+          waiter: SettingsPane.waitLoading,
+        });
+      },
+    );
+  });
+
+  after('Delete test data', () => {
+    cy.getAdminToken();
+    Users.deleteViaApi(user.userId);
+  });
+
+  it(
+    'C418594 Verify that final custom field can be removed (volaris) (TaaS)',
+    { tags: [TestTypes.extendedPath, DevTeams.volaris] },
+    () => {
+      cy.visit(TopMenu.customFieldsPath);
+      CustomFields.addCustomTextField(fieldData);
+      cy.wait(1000);
+      CustomFields.addCustomCheckBox(checkboxData);
+      cy.wait(1000);
+      CustomFields.addCustomSingleSelect(singleSelectData);
+      cy.wait(1000);
```

Comment:
In Step 5 need to verify that At least 3 custom fields are present on the 3rd pane view